### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by edernucci** [Azure/azure-documentdb-datamigrationtool#132]
According to the current model of creating open source projects adopted by Microsoft, it is expected by the community that tools are developed for multiple platforms.

Being forced to use a virtual machine for a cloud tool is unacceptable today.

**Comment by mrjamiebowman** [comment]
Yeah, I agree. I want to use this in a pipeline to back up our DEV databases so we can destroy everything at night and restore it in the morning.

**Comment by joelhulen** [comment]
Adding to the v1.9.0 milestone, which should provide a Linux-compatible console version.

**Comment by crpietschmann** [comment]
Reopening, closed on accident.

**Comment by sheeeng** [comment]
Perhaps #199?
  - [buildDebug.sh](https://github.com/Azure/azure-documentdb-datamigrationtool/pull/199/files#diff-ec518f905d1118c33a8b7bb6a9481527be8fe54ef9e73fbbc20690448994e1de)
  - [buildRelease.sh](https://github.com/Azure/azure-documentdb-datamigrationtool/pull/199/files#diff-daa70afcd870c3739811e0b07c4eff6a6f673b04e26e11d94525a3b2657f4b1b)
